stubs/requests/@tests/test_cases/check_post.py

```diff
@@ -44,8 +44,8 @@ def gen() -> Iterable[bytes]:
 requests.post("http://httpbin.org/anything", data="foobar").json()["data"]
 
 # Files
-requests.post("http://httpbin.org/anything", data=open("/tmp/foobar", "rb")).json()["data"]
-requests.post("http://httpbin.org/anything", data=open("/tmp/foobar", "r")).json()["data"]
+requests.post("http://httpbin.org/anything", data=open("/tmp/foobar", "rb", encoding="UTF-8")).json()["data"]
+requests.post("http://httpbin.org/anything", data=open("/tmp/foobar", "r", encoding="UTF-8")).json()["data"]
```

Comment:
The code in this file is never actually run (it's a test-case), but I guess there's no way of telling the flake8 plugin that, and I don't think this change does any actual harm :)